compiler/util/files.cpp

```diff
@@ -440,6 +440,18 @@ std::string runPrintChplEnv(std::map<std::string, const char*> varMap) {
   return runCommand(command);
 }
 
+std::string getChplPythonVersion() {
+  // Runs util/chplenv/chpl_python_version.py and left-trims the string
```

Comment:
I don't think this is left-triming, wouldn't that be taking characters off the left? I'd just say `Runs util/chplenv/chpl_python_version.py stripping newlines`